sway-lib-std/src/hash.sw

```diff
@@ -500,3 +500,122 @@ fn test_hasher_sha256_bytes() {
     let sha256 = hasher.sha256();
     assert(sha256 == 0x4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a);
 }
+
+// The hashes for the following test can be obtained in Rust by running the following script:
+// https://www.rustexplorer.com/b/kbg0oc
+
```

Comment:
```suggestion
```

This is a thoughtful addition; however, I think we should not be linking external sites in the source code. Particularly, if they do not belong to us and if someone may easily search for a site to replicate the hash of a value.

Reply:
Got it, that makes sense. 👍